src/main/java/org/activiti/cloud/runtime/RuntimeBundleMetaDataConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package org.activiti.cloud.runtime;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+
+import com.netflix.appinfo.ApplicationInfoManager;
+import org.springframework.beans.factory.BeanClassLoaderAware;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnClass(ApplicationInfoManager.class)
+public class RuntimeBundleMetaDataConfiguration implements BeanClassLoaderAware {
```

Comment:
Do we really need to need to implements `BeanClassLoaderAware`?

Reply:
not really.. let's remove that..